packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -54,58 +69,121 @@ export const ArticlePostCard = forwardRef(function PostCard(
     );
   }
 
+  useEffect(() => {
+    if (!hasEngagementLoopAccess || !hasBeenRead) {
+      return setShowEngagementLoop(false);
+    }
+
+    if (hidePostFeedback) {
+      return setShowEngagementLoop(false);
+    }
+
+    return setShowEngagementLoop(true);
+  }, [
+    hasBeenRead,
+    hasEngagementLoopAccess,
+    showEngagementLoop,
+    hidePostFeedback,
+  ]);
```

Comment:
We can probably replace this `useEffect` + `useState` pattern with `useMemo` only to compute inline. Saves on re-renders.